pipeline/molecular_phenotypes/calling/RNA_calling.ipynb

```diff
@@ -187,9 +187,19 @@
     "    --STAR-index reference_data/STAR_Index/ \\\n",
     "    --RSEM-index reference_data/RSEM_Index/ \\\n",
     "    --container container/rna_quantification.sif \\\n",
-    "    --mem 40G"
+    "    --mem 40G --is_stranded True"
```

Comment:
Is our data stranded? When preparing for the reference i didn't use stranded option ...

Also please use `--is-stranded` without `True` to flag this. it's a boolean variable in SoS

Reply:
Fixed, didnt realized some changes was not saved.